### ISSUE TITLE
log with data arrays not copying content

### ISSUE THREAD

**Issue by jorisbontje** [ethereum/serpent#34]
```
def test_log_data():
    log(data=[1,2,3])
```

This returns:
```
('LOG', 'c305c901078781c232a2a521c2af7980f8385ee9', [], [0, 0, 0])
```

instead of [1, 2, 3]

Full test code https://github.com/jorisbontje/sleth/blob/master/test/test_log.py

**Comment by vbuterin** [comment]
Fixed.

**Comment by jorisbontje** [comment]
Fixed in 0617417185644e12255b632d5d6bbf7e3977a4da
